tests/drivers/counter/counter_basic_api/boards/nrf54l20pdk_nrf54l20_cpuapp.overlay

```diff
@@ -0,0 +1,35 @@
+&timer00 {
+	prescaler = <6>;
+	status = "okay";
+	max-frequency = <DT_FREQ_M(64)>;
```

Comment:
If neeed it should be moved to dts/common/nordic/nrf54l_05_10_15.dtsi

Reply:
Moved to `nrf54l20.dtsi`